### ISSUE TITLE
Using Accessibility module with boost module causes all markers to show on zoom out

### ISSUE THREAD

**Issue by jasekiw** [highcharts/highcharts#17320]
#### Expected behaviour

Zooming in and zooming out should not cause all the markers to show on the graph when markers are disabled.

```
 plotOptions: {
        series: {
            marker: { enabled: false },
        }        
  },
```
#### Actual behaviour
Zooming in and zooming out causes all the points to have markers show on the zoomed out graph. 

Zoom in:
![image](https://user-images.githubusercontent.com/4037224/170042541-9ff5a58e-63f3-4490-aad4-1ef6ad4884c1.png)
Once you zoom out, all the markers will show.
Zoom out:
![image](https://user-images.githubusercontent.com/4037224/170042139-19b1e6ad-ca73-4faa-baf5-bebe753ab5c6.png)

#### Live demo with steps to reproduce
https://jsfiddle.net/bhurlston/wxc8q2bv/3/

#### Product version
10.1.0. This also seems to be an issue in 9.3.3. Most likely affects a great amount of versions.

#### Affected browser(s)
Tested in Chrome, latest version

**Comment by karolkolodziej** [comment]
Thank you for reporting that issue.

_FYI @oysteinmoseng  Internal note:_
Zooming in triggers the `seriesOnRender` then the `forceZeroOpacityMarkerOptions` is fired.
But the markers are not being reseted after zooming out.
Minimal demo: https://jsfiddle.net/BlackLabel/0eyL9dht/
https://github.com/highcharts/highcharts/blob/99479ccbcedd02ee7e67853877535cd873a71f12/ts/Accessibility/Components/SeriesComponent/ForcedMarkers.ts#L252-L273
Minimal demo: https://jsfiddle.net/BlackLabel/0eyL9dht/

**Comment by oysteinmoseng** [comment]
It seems the `Series.render` event is never fired for boosted series. Not really related to the a11y module then, unless there is a different event from Boost we should listen on?

**Comment by karolkolodziej** [comment]
The chart is not being rerendered because of that piece of code from the Chart `zoom` method:
https://github.com/highcharts/highcharts/blob/c11ce666605334e6ad245a8041f387f6e0fc3f87/ts/Core/Chart/Chart.ts#L3619-L3628

I think the `selection` or `zoom` events might be used or perhaps this `pick` should be modified somehow?

**Comment by jasekiw** [comment]
@karolkolodziej  Any update on this? I noticed it got labeled as stale. I'm commenting here to keep it from being closed.

**Comment by karolkolodziej** [comment]
@jasekiw Let me discuss it internally with the team and I will get back to you with some estimates 😉

**Comment by bre1470** [comment]
Internal note: Boost fires `renderedCanvas` event via `Series.renderCanvas` as `render` event is incompatible with boosted series states.

**Comment by jasekiw** [comment]
@karolkolodziej thanks for the update!

**Comment by karolkolodziej** [comment]
@jasekiw We will take a closer look in the upcoming week and will keep you posted!
